src/components/Snackbar/index.tsx

```diff
@@ -0,0 +1,177 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```

Comment:
Please remove the BMW line, it's not needed and adjust the year from 2023 to 2024 in the EF line.

Reply:
@evegufy Made the changes